fhttp/http_client.go

```diff
@@ -426,6 +434,16 @@ func NewStdClient(o *HTTPOptions) (*Client, error) {
 			if o.Resolve != "" {
 				addr = o.Resolve + addr[strings.LastIndex(addr, ":"):]
 			}
+			// TODO: Find out how many time this get called. Should be num of conn + error
+			conn, err := net.Dial(network, addr)
```

Comment:
? we can't connect and throw away the socket

Reply:
updated